src/pages/on-chain-governance/enum/proposalStatus.ts

```diff
@@ -0,0 +1,4 @@
+export enum proposalStatus {
```

Comment:
let's use pascal case for enum naming, and usually pascal case for its value too